.github/integrationBenchScripts/08_processResults.sh

```diff
@@ -0,0 +1,98 @@
+circuit=$(echo "$l" |  awk '{print $1}')
+time=$(date +%Y-%m-%d_%H-%M-%S)
+test_id=$time-$circuit-$degree-Benchmark
+
+cd "$trigger_results_dir"
+tar -czvf ./"$test_id".tar.gz ./*proverlog ./*.stats
+
+cp ../zkevm-circuits/.github/integrationBenchScripts/reporting*.py .
+sudo cp *proverlog /var/www/www_logs/
+proverlog="http://43.130.90.57/www_logs/"$(ls -t /var/www/www_logs | head -1)
+instance_commitments=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*instance.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $1}')
+instance_evaluations=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*instance.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $2}')
+advice_commitments=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*advice.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $1}')
+advice_evaluations=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*advice.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $2}')
+fixed_commitments=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*fixed.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $1}')
+fixed_evaluations=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*fixed.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $2}')
+lookups_commitments=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*lookups.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $1}')
+lookups_evaluations=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*lookups.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $2}')
+equality_commitments=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*equality.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $1}')
+equality_evaluations=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*equality.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $2}')
+vanishing_commitments=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*vanishing.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $1}')
+vanishing_evaluations=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*vanishing.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $2}')
+multiopen_commitments=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*multiopen.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $1}')
+multiopen_evaluations=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*multiopen.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $2}')
+polycomm_commitments=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*polycomm.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $1}')
+polycomm_evaluations=$(ls -t *proverlog | xargs cat | sed 's/\x1B\[[0-9;]*[JKmsu]//g' | egrep -o  "|.*polycomm.*" | egrep -o [0-9]+[\ ]+[0-9]+ | awk '{print $2}')
```

Comment:
Can this moved to a function to reduce the amount of deplication? This was automatically generated, no idea if it works:
```
extract_data() {
    local keyword=$1
    local column=$2
    ls -t *proverlog | xargs cat | \
    sed 's/\x1B\[[0-9;]*[JKmsu]//g' | \
    egrep -o "|.*$keyword.*" | \
    egrep -o '[0-9]+[\ ]+[0-9]+' | \
    awk -v col="$column" '{print $col}'
}

instance_commitments=$(extract_data "instance" 1)
instance_evaluations=$(extract_data "instance" 2)
advice_commitments=$(extract_data "advice" 1)
advice_evaluations=$(extract_data "advice" 2)
fixed_commitments=$(extract_data "fixed" 1)
fixed_evaluations=$(extract_data "fixed" 2)
lookups_commitments=$(extract_data "lookups" 1)
lookups_evaluations=$(extract_data "lookups" 2)
equality_commitments=$(extract_data "equality" 1)
equality_evaluations=$(extract_data "equality" 2)
vanishing_commitments=$(extract_data "vanishing" 1)
vanishing_evaluations=$(extract_data "vanishing" 2)
multiopen_commitments=$(extract_data "multiopen" 1)
multiopen_evaluations=$(extract_data "multiopen" 2)
polycomm_commitments=$(extract_data "polycomm" 1)
polycomm_evaluations=$(extract_data "polycomm" 2)
```